graylog2-web-interface/src/components/search/MessageShow.jsx

```diff
@@ -71,7 +71,6 @@ class MessageShow extends React.Component {
                          message={message}
                          inputs={inputs}
                          streams={streams}
-                         nodes={nodes}
                          renderForDisplay={this.renderForDisplay}
                          showTimestamp />
```

Comment:
`nodes` are currently nod provided by components which implement this component.

I did not migrate it to TS, because I want to keep the backport simple.